src/components/ui/ListEntry.js

```diff
@@ -11,16 +12,14 @@ const Icons = styled.div`
   }
 `
 const PrimaryText = styled.div`
```

Comment:
It would be more semantic for this text component (along with SecondaryText) be a `p` instead of a `div`

Reply:
The reason we used div was because p added tons of extra padding which messed up the spacing. Do you have ideas on how this could be fixed

Reply:
You can remove the padding.